README.md

```diff
@@ -31,5 +31,5 @@ The [project homepage](http://janusgraph.org) contains more information on Janus
 
 ## Contributing
 
-Please see [`CONTRIBUTING.md`](CONTRIBUTING.md) for more information, including
```

Comment:
@kottmann any updates on this? seems that this has gotten stalled. as @amcp mentions above, it seems like the files are in the wrong spot.